docassemble/ALKilnInThePlayground/data/static/alkiln_copy_button.css

```diff
@@ -1,56 +1,32 @@
 .accordion {
-  note: "Keeps the curved border even when children have their own backgrounds";
+  note: "Keeps the curved border even when children have their own backgrounds. Discuss: Should these styles be in the main css file?";
   contain: paint;
 }
-.accordion.output.generated {
-  note: "Generated output outer background and border styles";
-  border-radius: var(--bs-accordion-border-radius);
+.accordion {
+  note: "Accordion outer background and border styles";
+  border-radius: var(--bs-border-radius);
 }
 .generated .alkiln_copy {
-  background-color: var(--bs-accordion-active-bg);
   border: none;
 }
 .alkiln_copy .accordion-button:focus,
 .alkiln_copy .accordion-button:not(.collapsed) {
   box-shadow: none;
 }
-.alkiln_copy .accordion-header .row,
-.accordion-header .row {
-  margin: initial;
+.accordion-header:has(h3) {
+  padding-top: .2em;
+  padding-bottom: .2em;
+}
+.accordion-header .row:has(h3) {
+  margin-left: initial;
 }
 
 note {
   note: "Copy button and other relevant elements container. Discuss: Other sections having sticky headings.";
 }
-.alkiln_copy .sticky_scroll {
-  position: sticky;
-  z-index: 1;
-  note: "Puts this element below the nav bar to keep it visible. I'm not sure where that navbar padding is set. I haven't found it on GitHub. Element .da-pad-for-navbar sets the padding for the spacer that keeps the nav bar from covering the rest of the page.";
-  top: var(--nav-bar-height);
-}
 #console .alkiln_copy .sticky_scroll {
-  note: "Puts this element below the nav bar to keep it visible (it's got a transform pushing it up next to the heading, so it needs extra. Discuss: put heading and copy buttons in the same element). I'm not sure where that navbar padding is set. I haven't found it on GitHub. Element .da-pad-for-navbar";
-  top: calc( var(--nav-bar-height) + 2.75em);
-}
-
-.alkiln_copy .alk_grid {
-  display: grid;
-  gap: .4em;
-  note: "Ensure the combined width of all .alk_col:last-child items is the same in each grid of this type. Otherwise this sizing may look to wide or too thin, hiding other items in the column.";
-  grid-template-columns: 1fr max-content;
-  padding-top: .2em;
-  padding-bottom: .2em;
-  padding-right: .2em;
-}
-
-
-.alkiln_copy .alk_col {
-  font-size: .9em;
-  padding: 0;
-}
-
-.alk_col:last-child {
-  text-align: right;
+  note: "Puts this element below the nav bar to keep it visible (it's got a transform pushing it up next to the heading, so it needs extra. I'm not sure where that navbar padding is set. I haven't found it on GitHub. Element .da-pad-for-navbar";
+  top: calc( var(--detected-nav-bar-height) + 2.75em);
```

Comment:
Is this outdated now that we're calculating based on heights of h2/h3/etc detected with javascript?